src/PhpWord/Element/Image.php

```diff
@@ -503,31 +531,45 @@ private function setFunctions(): void
         switch ($this->imageType) {
             case 'image/png':
                 $this->imageCreateFunc = $this->sourceType == self::SOURCE_STRING ? 'imagecreatefromstring' : 'imagecreatefrompng';
-                $this->imageFunc = 'imagepng';
+                $this->imageFunc = function ($resource): void {
+                    imagepng($resource, null, $this->imageQuality);
+                };
                 $this->imageExtension = 'png';
+                $this->imageQuality = 0;
```

Comment:
PNG $quality is the zlib compression level, it is a lossless value. $filters is what enables any lossy compression but it is disabled by default. Setting PNG quality to 0 disables all compression resulting in larger files with no quality benefit.

The default for https://www.php.net/imagepng is -1 which in-turn defaults to the zlib compression default which from the zlib documentation is a quality value of 6 

> The compression level must be Z_DEFAULT_COMPRESSION, or between 0 and 9: 1 gives best speed, 9 gives best compression, 0 gives no compression at all (the input data is simply copied a block at a time). Z_DEFAULT_COMPRESSION requests a default compromise between speed and compression (currently equivalent to level 6).

Suggest changing the PNG quality value to -1 not 0 in this patch.